src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalService.kt

```diff
@@ -356,13 +332,14 @@ class CalculationTransactionalService(
     calculationUserInputs: CalculationUserInputs,
   ): CalculationBreakdown {
     if (previousCalculationResults.calculationType == CalculationType.CALCULATED) {
-      val (workingBooking, bookingCalculation) = calculationService.calculateReleaseDates(booking, calculationUserInputs)
-      if (bookingCalculation.dates == previousCalculationResults.dates) {
+      val calculationOutput = calculationService.calculateReleaseDates(booking, calculationUserInputs) // Validation stage 3 of 4
```

Comment:
Not sure this comment is meant to be here?

Reply:
lol yea. Copy and paste mistake!

Reply:
Fixed.